Library/Homebrew/dependency.rb

```diff
@@ -29,7 +29,31 @@ def hash
   end
 
   def to_formula
-    formula = Formulary.factory(name)
```

Comment:
I'll update these methods, thanks. There were two ways to handle this thing about renames:
  1. Updating `Formulary`.
  2. Changing  `Dependency#to_formula` behaviour.

I discussed that problem with @mikemcquaid after I had implemented it and we decided to change it according to other maintainer's feedback if necessary.